### PR TITLE
feat(clusterator): Add create and delete cluster jobs

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: jobs/clusterator_jobs.groovy [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      string("GCLOUD_CREDENTIALS", "15122437-45b8-48ac-bb84-652394c8a927")
+    }
+  }
+
+  steps {
+    shell """
+      #!/usr/bin/env bash
+
+      set -eo pipefail
+      docker run -e GCLOUD_CREDENTIALS=\${GCLOUD_CREDENTIALS} quay.io/jchauncey/clusterator:git-d3bb8c6
```
Comment (vdice) [inline jobs/clusterator_jobs.groovy]: does this command need `create` at the end?

File: jobs/clusterator_jobs.groovy [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  triggers {
+    cron('0 7 1 * 1-5')
+  }
+
+  parameters {
+    stringParam('NUMBER_OF_CLUSTERS', '10', 'Number of clusters to create at 1 time')
+    stringParam('NUM_NODES', '5', 'Number of nodes in each cluster')
+    stringParam('MACHINE_TYPE', 'n1-standard-4', 'Node type')
+    stringParam('VERSION', '1.2.5', 'The version of kubernetes to use.')
+    stringParam('sha1', 'master', 'Specific Git SHA to test')
```
Comment (vdice) [inline jobs/clusterator_jobs.groovy]: Is this param being used?

File: jobs/clusterator_jobs.groovy [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      string("GCLOUD_CREDENTIALS", "15122437-45b8-48ac-bb84-652394c8a927")
+    }
+  }
+
+  steps {
+    shell """
+      #!/usr/bin/env bash
+
+      set -eo pipefail
+      docker run -e GCLOUD_CREDENTIALS=\${GCLOUD_CREDENTIALS} quay.io/jchauncey/clusterator:git-d3bb8c6 delete
```
Comment (vdice) [inline jobs/clusterator_jobs.groovy]: Do we want the ability to override the `clusterator` image tag?  And/or would we rather use `canary`/`latest` here (after explicitly running `docker pull` beforehand to bust the cache)?

File: jobs/clusterator_jobs.groovy (hunk 1 above)
Comment (jchauncey) [inline jobs/clusterator_jobs.groovy]: no create is the default

File: jobs/clusterator_jobs.groovy (hunk 2 above)
Comment (jchauncey) [inline jobs/clusterator_jobs.groovy]: no forgot to remove it

File: jobs/clusterator_jobs.groovy (hunk 3 above)
Comment (jchauncey) [inline jobs/clusterator_jobs.groovy]: my thought was to use the immutable image instead of latest

Comment (jchauncey) [inline jobs/clusterator_jobs.groovy]: I want to use immutable images if possible.

**Pull request merged**